### PR TITLE
Add methods get and count in storage

### PR DESCRIPTION
**Pull request by diesgaro**
I added the next methods in FileStorage and DBStorage Classes:

get(): method that retrieve a  object  by class name and id
count(): method that count the number of objects in storage by class name or all objects

### CODE REVIEW

**Comment by KevinCastroP** [comment]
Is better if you make a short comments inside your code

File: models/engine/db_storage.py [hunk 1]
```diff
@@ -74,3 +74,33 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """
+        Function that retrieve one object
```
Comment (KevinCastroP) [inline models/engine/db_storage.py:80]: This comment be more short

File: models/engine/file_storage.py [hunk 2]
```diff
@@ -68,3 +68,40 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """
+        Function that retrieve one object
```
Comment (KevinCastroP) [inline models/engine/file_storage.py:74]: This comment can be more shorts

File: models/engine/db_storage.py (hunk 1 above)
Comment (KevinCastroP) [inline models/engine/db_storage.py:80]: This comment can be more short

Comment (diesgaro) [reply models/engine/db_storage.py:80]: changes apply

**Comment by KevinCastroP** [comment]
Good job! now your changes going move it to the master branch

File: models/engine/file_storage.py (hunk 2 above)
Comment (diesgaro) [reply models/engine/file_storage.py:74]: changes apply

**Pull request merged**